docker-image-src/4.0/docker-entrypoint.sh

```diff
@@ -206,6 +206,34 @@ function install_neo4j_labs_plugins
   rm "${_old_config}"
 }
 
+function process_ssl_policies
+{
+    local _found_a_policy_scope=false
+    local _ssl_policy_scopes=("bolt" "https" "cluster" "backup" "fabric")
+    for scope in ${_ssl_policy_scopes[@]}
+    do
+        echo $scope
+        # does this policy folder exist in dir
+        if [[ -d /ssl/$scope ]]; then
+            local _connector_flag=NEO4J_dbms_connector_${scope}_enabled
+            : ${!_connector_flag:=true}
```

Comment:
according to the docs `dbms.connector.<foo>` is only available for `bolt` `http` and `https`.
If we do this for the others I presume we will get warnings logged about unrecognised config values